packages/sdk-react/src/components/onboarding/transformers/generateOnboardingValidationSchema.ts

```
@@ -69,34 +67,40 @@ export const generateOnboardingValidationSchema = ({
+        const validator =
+          schemaKey in schema && (schema[schemaKey] as ValidatorType);
 
-      const subresourceType = getSubresourceTypeByKey(key);
+        const subresourceType = getSubresourceTypeByKey(key);
 
-      if (!subresourceType && validator) {
-        return { ...acc, ...{ [key]: getValidatorSettings(field, validator) } };
-      }
+        if (!subresourceType && validator) {
+          return {
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)